.eslintrc

```diff
@@ -75,7 +75,8 @@
     "DEV": true,
     "fixture": true,
     "test": true,
-    "page": true
+    "page": true,
+    "__VERSION_INFO__": true
```

Comment:
ah! Cool hack, didn't know this was the cause for many of the eslint annotations!